content/learning-paths/laptops-and-desktops/mca-godbolt/running_mca.md

```diff
@@ -0,0 +1,350 @@
+[5,1]     .D=========================eER.    .    .    .    . .   add   x1, x1, x3
+[5,2]     .D==========================eER    .    .    .    . .   add   x1, x1, x4
+[5,3]     .D===========================eER   .    .    .    . .   add   x1, x1, x5
+[5,4]     .D============================eER  .    .    .    . .   add   x1, x1, x6
+[6,0]     .D=============================eER .    .    .    . .   add   x1, x1, x2
+[6,1]     .D==============================eER.    .    .    . .   add   x1, x1, x3
+[6,2]     . D==============================eER    .    .    . .   add   x1, x1, x4
+[6,3]     . D===============================eER   .    .    . .   add   x1, x1, x5
+[6,4]     . D================================eER  .    .    . .   add   x1, x1, x6
+[7,0]     . D=================================eER .    .    . .   add   x1, x1, x2
+[7,1]     . D==================================eER.    .    . .   add   x1, x1, x3
+[7,2]     . D===================================eER    .    . .   add   x1, x1, x4
+[7,3]     . D====================================eER   .    . .   add   x1, x1, x5
+[7,4]     . D=====================================eER  .    . .   add   x1, x1, x6
+[8,0]     . D======================================eER .    . .   add   x1, x1, x2
+[8,1]     . D=======================================eER.    . .   add   x1, x1, x3
+[8,2]     . D========================================eER    . .   add   x1, x1, x4
+[8,3]     . D=========================================eER   . .   add   x1, x1, x5
+[8,4]     . D==========================================eER  . .   add   x1, x1, x6
+[9,0]     . D===========================================eER . .   add   x1, x1, x2
+[9,1]     . D============================================eER. .   add   x1, x1, x3
+[9,2]     . D=============================================eER .   add   x1, x1, x4
+[9,3]     .  D=============================================eER.   add   x1, x1, x5
+[9,4]     .  D==============================================eER   add   x1, x1, x6
+
+
+Average Wait times (based on the timeline view):
+[0]: Executions
+[1]: Average time spent waiting in a scheduler's queue
+[2]: Average time spent waiting in a scheduler's queue while ready
+[3]: Average time elapsed from WB until retire stage
+
+      [0]    [1]    [2]    [3]
+0.     10    22.6   0.1    0.0       add        x1, x1, x2
+1.     10    23.5   0.0    0.0       add        x1, x1, x3
+2.     10    24.4   0.0    0.0       add        x1, x1, x4
+3.     10    25.3   0.0    0.0       add        x1, x1, x5
+4.     10    26.3   0.0    0.0       add        x1, x1, x6
+       10    24.4   0.0    0.0       <total>
+```
+
+These states are represented by the following characters:
+- D : Instruction dispatched.
+- e : Instruction executing.
+- E : Instruction executed.
+- R : Instruction retired.
+- = : Instruction already dispatched, waiting to be executed.
+- \- : Instruction executed, waiting to be retired.
+
```

Comment:
I believe the reader would benefit from a quick explanation on how to read the timeline view. A quick word on what the index is would definitely help a lot.

Reply:
I can explain what the index is. In terms of reading the timeline view, I'll see what I can say about that, but I'm reluctant to repeat what is already in the MCA documentation.

Reply:
You definitely should not repeat the MCA documentation, this is about giving enough hints to the reader that he can at least vaguely understand the timeline view. Personal example: I have had a hard time to understand it, although I have worked quite a lot in LLVM and looked into scheduling (a long time ago), in LLVM or using other models --- I had to dive into the MCA manual to make sense out of the view.